Source/Core/DolphinQt2/DolphinQt2.vcxproj

```diff
@@ -40,8 +40,9 @@
   </PropertyGroup>
   <ItemDefinitionGroup>
     <Link>
-      <AdditionalLibraryDirectories>$(ExternalsDir)ffmpeg\lib;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories>$(ExternalsDir)ffmpeg-win64\lib;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
```

Comment:
I'm almost certain that we don't need "$(ExternalsDir)ffmpeg-win64\lib;" anymore now that we're dynamically linking.